### DISCUSSION TITLE
How to return to last Survey item

### DISCUSSION THREAD

**Discussion by Chenghao-Zhou** [jspsych/jsPsych#3002] [Q&A] [answered]
Hi,

The parameter `button_label`, is the label of the button to end the trial and go to the next question. I am wondering how can I return to the previous question by clicking one button. For example, If I have 12 Survey items (Likert scale of 5) and put 3 on each page, then there will be 4 pages. I want the subject to have two choices (continue and return to the previous page) before completing the fourth page and clicking end.

**Comment by ChristopheBossens** [comment, marked as answer]
Hi,

I think the survey plugin already allows you to do that? There is a multipage example in de code demos [here](https://www.jspsych.org/7.3/plugins/survey/#examples).

**Reply by Chenghao-Zhou** [reply]
haha thanks!! I used a very old version and I will update it.
